src/app/static/src/tests/components/vuex/runReport/gitSelections.test.ts

```diff
@@ -93,7 +90,7 @@ describe("GitSelections", () => {
         expect(wrapper.find("#git-commit-form-group").exists()).toBe(false);
     });
 
-    it("first git branch is selected automatically and selecting a git branch triggers select branch action", async () => {
+    it("selecting a git branch triggers select branch action", () => {
```

Comment:
Might be wrong but I don't think there's a test in this file that the `selectedCommit` and `seleectedBranch` from the store are the values in selects (just that the correct options are rendered)?